### ISSUE TITLE
(Trezor T) trezor-agent ssh doesn't work with ed25519 public key

### ISSUE THREAD

**Issue by AgentME** [romanz/trezor-agent#229]
With my Trezor T, if I use the `-e ed25519` option with trezor-agent, I get an error:

```
$ trezor-agent -e ed25519 -c exampleserver.foo
2018-04-03 19:44:43,866 WARNING      error: invalid ed25519 public key: b'\x03\xb1h\xad\x8a,\xb7\xa8M\xf2\xe55\xe6\x94\xaezv4\xb0r\xfe[\x961,\x1d\xc0\xfc\xcb\xbe\x06\xc5\xe0' [server.py:63]
Traceback (most recent call last):
  File "/home/chris/.local/lib/python3.5/site-packages/libagent/server.py", line 58, in handle_connection
    reply = handler.handle(msg=msg)
  File "/home/chris/.local/lib/python3.5/site-packages/libagent/ssh/protocol.py", line 103, in handle
    reply = method(buf=buf)
  File "/home/chris/.local/lib/python3.5/site-packages/libagent/ssh/protocol.py", line 111, in list_pubs
    keys = self.conn.parse_public_keys()
  File "/home/chris/.local/lib/python3.5/site-packages/libagent/ssh/__init__.py", line 205, in parse_public_keys
    for pk in self.public_keys()]
  File "/home/chris/.local/lib/python3.5/site-packages/libagent/ssh/__init__.py", line 199, in public_keys
    self.public_keys_cache = conn.export_public_keys(self.identities)
  File "/home/chris/.local/lib/python3.5/site-packages/libagent/ssh/client.py", line 28, in export_public_keys
    curve_name=i.curve_name)
  File "/home/chris/.local/lib/python3.5/site-packages/libagent/formats.py", line 152, in decompress_pubkey
    raise ValueError(msg)
ValueError: invalid ed25519 public key: b'\x03\xb1h\xad\x8a,\xb7\xa8M\xf2\xe55\xe6\x94\xaezv4\xb0r\xfe[\x961,\x1d\xc0\xfc\xcb\xbe\x06\xc5\xe0'
chris@exampleserver.foo's password: 

```

Works fine with my classic trezor.

I'm on Ubuntu 16.04 with trezor-agent version 0.9.2. I updated it and its dependencies right before trying this.

**Comment by romanz** [comment]
This issue should be resolved at 2.0.6 TREZOR model T firmware.
It can be downloaded from https://github.com/trezor/webwallet-data/tree/master/firmware/2.

**Comment by AgentME** [comment]
Thanks, it works!
